install-script/install.ps1

```diff
@@ -346,11 +354,59 @@ function Get-BinaryFromURI {
     Write-Host "Downloaded ${Path}"
 }
 
-##
+function Send-Installation-Logs {
+    param (
+        [Parameter(Mandatory, Position=0)]
+        [HttpClient] $HttpClient,
+
+        [Parameter(Mandatory, Position=1)]
+        [string] $Endpoint,
+
+        [Parameter(Mandatory, Position=2)]
+        [string] $Path
+    )
+
+    $Content = Get-Content -Path $Path
+    $StringContent = [System.Net.Http.StringContent]::new($Content)
+
+    $response = $HttpClient.PostAsync($Endpoint, $StringContent).GetAwaiter().GetResult()
+    if (!($response.IsSuccessStatusCode)) {
+        $statusCode = [int]$response.StatusCode
+        $reasonPhrase = $response.StatusCode.ToString()
+        $errMsg = "${statusCode} ${reasonPhrase}"
+
+        if ($response.Content -ne $null) {
+            $content = $response.Content.ReadAsStringAsync().GetAwaiter().GetResult()
+            $errMsg += ": ${content}"
+        }
+
+        Write-Error -Message $errMsg
+    }
+}#
+#
 # Main code
 ##
 
 try {
+    $InstallationLogFile = New-TemporaryFile
+
+    if ($InstallationLogFileEndpoint -eq "") {
+        ## https://sumologic.atlassian.net/wiki/spaces/MAIN/pages/1601341782/Production+Deployments+and+More#ProductionDeploymentsandMore-StagingDeployments
```

Comment:
As much as this is useful for us, you should probably drop that comment since this is a public repo.